nginx/Dockerfile-alpine

```diff
@@ -30,18 +48,14 @@ COPY src/opt/modsecurity/activate-rules.sh /opt/modsecurity/
 COPY src/etc/modsecurity.d/*.conf /etc/modsecurity.d/
 COPY nginx/docker-entrypoint.sh /
 COPY nginx/nginx.conf /etc/nginx/nginx.conf
+COPY --from=release /opt/owasp-crs /opt/owasp-crs
 
-# hadolint ignore=DL3008,SC2016
+# Alpine needs sed because busybox does not support 'z' param
```

Comment:
```suggestion
# Alpine needs GNU 'sed' because the 'sed' program shipped with busybox does not support 'z' parameter for separating lines with a 'NUL' character.
```

Reply:
Weird, if you take a look at the files, this change was applied... 🤔

Reply:
Oh... sorry. Maybe I was looking at an older commit somehow...